web/src/components/TraceDrawer/TraceDrawer.tsx

```diff
@@ -0,0 +1,65 @@
+import {Drawer} from 'antd';
```

Comment:
The drawer might contain different things depending on the context, right now we display both the assertion form and the test results depending on the context